src/web-components/components/ContractNameForm/ContractNameForm.astro

```diff
@@ -0,0 +1,3 @@
+<script hoist>
```

Comment:
This is how a `js/ts` file that defines a `web-component` gets used as an `astro template`